stdlib/3/typing.pyi

```diff
@@ -302,6 +312,9 @@ class AbstractSet(_Collection[_T_co], Generic[_T_co]):
     def __xor__(self, s: AbstractSet[_T]) -> AbstractSet[Union[_T_co, _T]]: ...
     # TODO: Argument can be a more general ABC?
     def isdisjoint(self, s: AbstractSet[Any]) -> bool: ...
+    # Implement Sized (but don't have it as a base class).
+    @abstractmethod
+    def __len__(self) -> int: ...
```

Comment:
This is inherited from `_Collection`, so this seems unnecessary.

Reply:
Ah. Done.